### ISSUE TITLE
Care Card (immediate) - Details component text colour

### ISSUE THREAD

**Issue by davidhunter08** [nhsuk/nhsuk-frontend#475]
## What is the issue?

The text colour of the details component does not have sufficient colour contrast when used in a Care Card - Immediate.

<img width="660" alt="Screenshot 2019-06-24 at 10 52 27" src="https://user-images.githubusercontent.com/14331000/60009995-cbb95300-966e-11e9-849c-66d1bbae9d60.png">

## Solution

```
.nhsuk-care-card--immediate {

  .nhsuk-details,
  .nhsuk-details__summary {
    color: $color_nhsuk-white;
  }

  .nhsuk-details__summary {
    &:hover,
    &:focus {
      color: $color_nhsuk-black;
    }
  }

}
```

**Comment by AdamChrimes** [comment]
<img width="655" alt="Screenshot 2019-07-11 at 10 36 22" src="https://user-images.githubusercontent.com/11615501/61040238-1d304480-a3c8-11e9-8e83-ab1c41cb1a49.png">

**Comment by AdamChrimes** [comment]
Fixed in v2.3.0.
